msal/src/test/java/com/microsoft/identity/client/e2e/tests/network/nativeauth/NativeAuthPublicClientApplicationAbstractTest.kt

```diff
@@ -84,19 +73,13 @@ abstract class NativeAuthPublicClientApplicationAbstractTest : IPublicClientAppl
         return "" // Not needed for native auth flows
     }
 
-    abstract val defaultConfigType: ConfigType
-
     @Before
     open fun setup() {
         context = ApplicationProvider.getApplicationContext()
         activity = Mockito.mock(Activity::class.java)
         Mockito.`when`(activity.applicationContext).thenReturn(context)
-        Logger.getInstance().setEnableLogcatLog(true)
```

Comment:
Why were these removed?

Reply:
Because of reason here https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/2213#discussion_r1858467717

Reply:
I was asking about the logging itself, I don't see it happening anywhere else, unless I am wrong? Is the one in the real public client application active now?

Reply:
Since each test case now runs an application, the attached logging will cause conflicts.